packages/shared/src/components/cards/ArticlePostCard.tsx

```diff
@@ -57,54 +64,92 @@ export const ArticlePostCard = forwardRef(function PostCard(
   return (
     <FeedItemContainer
       {...props}
-      className={getPostClassNames(post, className, 'min-h-[22.5rem]')}
+      className={getPostClassNames(
+        post,
+        classNames(
+          className,
+          showFeedback && '!p-0',
+          !showFeedback && hasBeenRead && styles.read,
+        ),
+        'min-h-[22.5rem]',
+      )}
       style={{ ...style, ...customStyle }}
       ref={ref}
       flagProps={{ pinnedAt, trending }}
     >
       <CardButton title={post.title} onClick={onPostCardClick} />
-      <CardTextContainer>
-        <PostCardHeader
-          openNewTab={openNewTab}
-          source={post.source}
-          postLink={post.permalink}
-          onMenuClick={(event) => onMenuClick?.(event, post)}
-          onReadArticleClick={onReadArticleClick}
-        />
-        <CardTitle>{post.title}</CardTitle>
-      </CardTextContainer>
-      <Container className="mb-8 tablet:mb-0">
-        <CardSpace />
-        <PostMetadata
-          createdAt={post.createdAt}
-          readTime={post.readTime}
-          className="mx-4"
-        />
-      </Container>
-      <Container>
-        <PostCardFooter
-          insaneMode={insaneMode}
-          openNewTab={openNewTab}
-          post={post}
-          showImage={showImage}
-          onReadArticleClick={onReadArticleClick}
-        />
-        <ActionButtons
-          openNewTab={openNewTab}
-          post={post}
-          onUpvoteClick={onUpvoteClick}
-          onCommentClick={onCommentClick}
-          onBookmarkClick={onBookmarkClick}
-          onShare={onShare}
-          onShareClick={onShareClick}
-          onMenuClick={(event) => onMenuClick?.(event, post)}
-          onReadArticleClick={onReadArticleClick}
-          className={classNames(
-            'mx-4 justify-between',
-            !showImage && 'my-4 laptop:mb-0',
+
+      {showFeedback && <FeedbackCard post={post} />}
+
+      <ConditionalWrapper
+        condition={showFeedback}
+        wrapper={(wrapperChildren) => (
+          <div
+            className={classNames(
+              'p-2 border !border-theme-divider-tertiary rounded-2xl overflow-hidden',
+              styles.post,
+              styles.read,
+            )}
+          >
+            {wrapperChildren}
+          </div>
+        )}
+      >
+        <CardTextContainer>
+          {!showFeedback && (
+            <PostCardHeader
+              openNewTab={openNewTab}
+              source={post.source}
+              postLink={post.permalink}
+              onMenuClick={(event) => onMenuClick?.(event, post)}
+              onReadArticleClick={onReadArticleClick}
+            />
+          )}
+          <CardTitle className={classNames(showFeedback && '!line-clamp-2')}>
```

Comment:
Can we maybe make `line-clamp` optional/dynamic to allow no overwrites?

Reply:
Tried something in 2c47c833d224c19d63cfd8418c1becd9974e1aee let me know what you think 2c47c833d224c19d63cfd8418c1becd9974e1aee